tests/datm_conf/cpld_datm_mom6_cice5_cfsr.IN

```diff
@@ -20,5 +20,9 @@ cp    @[RTPWD]/CICE_IC/cice5_model_0.25.cpc*.nc cice5_model.res.nc
 
 # WARM START
 if [[ ! $MED_restart_data == '' ]]; then
+  if [[ @[CREATE_BASELINE] == 'true' ]]; then
+  cp  @[NEW_BASELINE]/MEDIATOR_CFSR/mediator_* .
```

Comment:
I think you also need 2 more changes: 1) add test dependency in rt.conf 2) copy the cold start test mediator restart files to NEW_BASELINE directory when the cold start finishes.

Reply:
Right now, the mediator files from cold-start runs are copied to NEW_BASELINE directory after the cold-start runs finish.  The dependency feature will be added.

Reply:
Thanks, Bin. This works.